pages/common/gyb.md

```diff
@@ -0,0 +1,20 @@
+# Got Your Back - GYB
+
+> Command line tool for backing up your Gmail messages to your computer using Gmail's API over HTTPS.
```

Comment:
We try to avoid any personal pronouns like `you`. How about this instead:
```suggestion
> Command line tool for locally backing up Gmail messages using Gmail's API over HTTPS.
```